task/control-plane-agent/src/mgs_gimlet/host_phase2.rs

```diff
@@ -120,7 +120,7 @@ impl HostPhase2Requester {
             State::NeedToSendFirstMgs(port) => {
                 current.state = State::WaitingForFirstMgs {
                     port,
-                    deadline: now + DELAY_TRY_OTHER_MGS,
+                    deadline: now.saturating_add(DELAY_TRY_OTHER_MGS),
```

Comment:
**nitpick:** The nice comments about why saturating add is okay to use are missing :(